partner_contact_birthdate/views/res_partner.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+     Copyright 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. -->
+
+<odoo>
+    <record id="view_personal_information_birthdate_date" model="ir.ui.view">
```

Comment:
The convention is to use the same id as in the inherited from module, but give a distinct name. Now the partner_personal_information_page did not follow this convention, so you have a choice of using personal_information here, or the original id. I would prefer the original id view_partner_form.

Reply:
Good idea.

Reply:
@NL66278 
Edited id to original id

Thank you for your review.